src/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_DurationScreen/durationPicker/DurationPicker.tsx

```diff
@@ -0,0 +1,252 @@
+import {Platform, View} from 'react-native';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import SvgDate from '@atb/assets/svg/mono-icons/time/Date';
+import {ThemeText} from '@atb/components/text';
+import {TicketAssistantTexts, useTranslation} from '@atb/translations';
+import RNDateTimePicker from '@react-native-community/datetimepicker';
+import {addDays, format} from 'date-fns';
+import {
+  dateDiffInDays,
+  getSliderIndex,
+  getDurationText,
+} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_DurationScreen/utils';
+import {SectionSeparator} from '@atb/components/sections';
+import {Slider} from '@atb/components/slider';
+import React, {useState} from 'react';
+import {StyleSheet} from '@atb/theme';
+import {useAccessibilityContext} from '@atb/AccessibilityContext';
+import {useLocaleContext} from '@atb/LocaleProvider';
+import {useTicketAssistantState} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistantContext';
+import {Button} from '@atb/components/button';
+
+type DurationPickerProps = {
+  duration: number;
+  setDuration: (duration: number) => void;
+};
+const currentDate = new Date();
+const durations = [1, 2, 3, 4, 5, 6, 7, 14, 21, 30, 60, 90, 120, 150, 180];
```

Comment:
Could this be injected from firestore to be customizable in the future?

Reply:
Would be valuable to have it since other companies would use it and should be aligned to their requirements, :) maybe another point of view from the team? @gorandalum what do you think?

Reply:
Might be better in Firestore, but I think quite a bit of work is necessary for other transport companies to use this feature, so we can tackle it then.